ledger-tool/src/main.rs

```diff
@@ -757,6 +772,22 @@ fn load_bank_forks(
             ..SnapshotConfig::default()
         })
     };
+
+    if let Some(halt_slot) = process_options.halt_at_slot {
+        for slot in starting_slot..=halt_slot {
+            if let Ok(Some(slot_meta)) = blockstore.meta(slot) {
+                if !slot_meta.is_full() {
+                    eprintln!("blockstore slot {} is not full which is required for replaying slots {}..={}",
+                        slot, starting_slot, halt_slot);
+                    exit(1);
+                }
+            } else {
+                eprintln!("blockstore missing data for slot {} which is required for replaying slots {}..={}", slot, starting_slot, halt_slot);
```

Comment:
nit - Similar to above:
"Unable to process from slot {} to {} due to blockstore missing slot {}"

Reply:
and done